cmake.deps/cmake/BuildLibtermkey.cmake

```diff
@@ -4,8 +4,6 @@ if(WIN32)
       ${DEPS_BUILD_DIR}/src/libtermkey/CMakeLists.txt
     COMMAND ${CMAKE_COMMAND} ${DEPS_BUILD_DIR}/src/libtermkey
       -DCMAKE_INSTALL_PREFIX=${DEPS_INSTALL_DIR}
-      # Pass toolchain
-      -DCMAKE_TOOLCHAIN_FILE=${TOOLCHAIN}
```

Comment:
Just to confirm, does the blame / git log show that these TOOLCHAIN lines were added in a commit related to cross-compiling?

Reply:
The git blame doesn't really show anything other than a generic "added dependency". But toolchains are for cross-compilation. Also we don't have any toolchains to pass anymore since they were deleted.